crates/forge/tests/e2e/running.rs

```diff
@@ -440,6 +440,37 @@ fn exit_first_flag_takes_precedence() {
         "#});
 }
 
+#[test]
+fn init_new_project_test() {
+    let temp = TempDir::new().unwrap();
+
+    let snapbox = runner();
+    snapbox
+        .current_dir(&temp)
+        .arg("--init")
+        .arg("test_name")
+        .assert()
+        .success();
+
+    let snapbox = runner();
+    snapbox
+        .current_dir(temp.child(Path::new("test_name")))
+        .arg("--exit-first")
```

Comment:
nit, but this arg shouldn't be passed here

Reply:
I added it because it will run fewer tests if any fail and it will break the test. I can remove it